review/src/OrderedRanges.elm

```diff
@@ -18,6 +18,24 @@ rule =
 expressionVisitor : Node Expression -> List (Error {})
 expressionVisitor node =
     case Node.value node of
+        Expression.Application [ Node _ (Expression.FunctionOrValue [] "Application"), Node _ (Expression.ListExpr ((Node firstItemRange _) :: _)) ] ->
+            let
+                { start, end } =
+                    Node.range node
+            in
+            if start.row == end.row then
+                [ Rule.errorWithFix
+                    { message = "Split Application on multiple likes"
```

Comment:
Nice rule 💯 
```suggestion
                    { message = "Split Application on multiple lines"
```

Reply:
Good catch :+1: 
And I had good inspiration :wink: 
I wonder what other node we can apply this to, I don't remember what I usually did manually.